static/js/feedback_widget.js

```diff
@@ -130,7 +130,11 @@ exports.show = function (opts) {
     meta.hide_me_time = new Date().getTime() + 4000;
 
     meta.$container.find('.feedback_title').text(opts.title_text);
-    meta.$container.find('.feedback_undo').text(opts.undo_button_text);
+
+    if (opts.undo_button_text) {
+        // Some actions like the n and p hotkeys don't need an undo option.
+        meta.$container.find('.feedback_undo').text(opts.undo_button_text);
```

Comment:
I think you'll need to actually explicitly hide and show the button here, rather than in the template.  You'll want to test going back and forth between the n/p feedback and some other thing that uses feedback.

Reply:
Hrm, I think we're ok here after looking more at the code.